node/src/bin/manager.rs

```diff
@@ -1391,6 +1428,31 @@ async fn main() -> anyhow::Result<()> {
                         }
                     }
                 }
+                Rewind {
+                    chain_name,
+                    block_hash,
+                    block_number,
+                    sleep,
+                    force,
+                    start_block,
+                } => {
+                    let notification_sender = ctx.notification_sender();
+                    let (store, primary) = ctx.store_and_primary();
+                    let deployments = vec![DeploymentSearch::AllOnChain { chain_name }];
+
+                    commands::rewind::run(
```

Comment:
Correct me if i'm wrong: But the logic here seems to be that we get all the deployments on a chain and then loop over it and rewind them one by one? 
That might be an issue when we have 1000's of deployments and the process would just run for hours.
And if it get cancelled midway, we'd need to start over.

In my opinion we need some way to manage this large number of deployments. Maybe this process should run in the background and we can come back and check the progress with another command. Something like what `graphman copy` has